### ISSUE TITLE
$(awk '{if($i==${values_Of_new_record[i]}) print "Primary key value is exist please try again";insertRecord }'  "$tableName")

### ISSUE THREAD

**Issue by shimaa829** [koalaman/shellcheck#1808]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2091, SC2154
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
$(awk '{if($i==${values_Of_new_record[i]}) print "Primary key value is exist please try again";insertRecord }'  "$tableName")
```

#### Here's what shellcheck currently says:
```
Line 2	SC2091: Remove surrounding $() to avoid executing output.
Line 2	SC2154: tableName is referenced but not assigned.
```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output

**Comment by gromgit** [comment]
Both those warnings are valid for your code excerpt. What exactly are you expecting to happen?
